inspirehep/modules/forms/validators/simple_fields.py

```diff
@@ -177,6 +177,27 @@ def get_content_type(url):
         raise StopValidation(message)
```

Comment:
Yes, sometimes we get errors because of that. A more robust check is pending to be implemented https://github.com/inspirehep/inspire-next/issues/1057